pyccel/parser/semantic.py

```diff
@@ -2561,8 +2561,8 @@ def _visit_FunctionDef(self, expr, **settings):
             # to the body of the function
             body = self.garbage_collector(body)
 
-            args    = [self.get_variable(a.name) if isinstance(a, Variable) else self.get_function(str(a.name)) for a in args]
-            results = list(OrderedDict((a.name,self.get_variable(a.name)) for a in results).values())
+            args    = list(args)
```

Comment:
Isn't `args` already a list? Is this line necessary?

Reply:
I think its a tuple let me check

Reply:
https://github.com/pyccel/pyccel/blob/master/pyccel/parser/semantic.py#L2462

Reply:
ah true sorry